csrc/preseg_passes/propagate_shardings.cpp

```diff
@@ -299,8 +279,7 @@ void PropagateShardingsPass::runPass(Fusion* fusion) {
         /*ref=*/ref_output,
         /*tvs=*/sharding_candidates,
         /*did_pos=*/did_pos,
-        /*allow_c2p=*/true,
-        /*allow_p2c=*/false);
+        /*direction=*/PropagationDirection::Backward);
```

Comment:
```suggestion
        PropagationDirection::Backward);
```
https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

You don't have to write function argument comments in this case because propagateDIDTransform takes arguments of different types, and a wrong order will likely fail C++'s type check.